polaris-react/src/components/Box/Box.tsx

```diff
@@ -168,12 +171,10 @@ export interface BoxProps {
    * paddingInlineEnd={{xs: '2', sm: '3', md: '4', lg: '5', xl: '6'}}
    */
   paddingInlineEnd?: Spacing;
-  /** Shadow */
+  /** Shadow color on box */
```

Comment:
```suggestion
  /** Shadow on box */
```
Should this just be `Shadow on box`?

Reply:
Ah yup that was a typo on my part.